bokeh/tile_providers.py

```diff
@@ -14,6 +14,17 @@
     )
 )
 
+#: Provide toner background tiles from http://tile.stamen.com
+STAMEN_TONER_BACKGROUND = WMTSTileSource(
+    url='http://tile.stamen.com/toner-background/{Z}/{X}/{Y}.png',
```

Comment:
Would be good to add that information to the comment so that it's in the docs.

Would also be good to add (presumably in the docstring at the top of this file) a link to docs for stamen that we can monitor for changes to their api.